app/shared/availability-view/TimelineGroups/TimelineGroup/AvailabilityTimeline/Reservation.js

```diff
@@ -28,7 +28,7 @@ function Reservation(props) {
         {startTime.format('HH:mm')} - {endTime.format('HH:mm')}
       </div>
       <div><FontAwesome fixedWidth name="user" /> {props.reserverName}</div>
-      <div><FontAwesome fixedWidth name="users" /> {props.numberOfParticipants}</div>
+      <div><span className="chair-icon" /> {props.numberOfParticipants}</div>
```

Comment:
- [ ] This could still be the user icon because it represents the number of participants. The chair icon is only meant to be used to indicate the capacity of resource. The name of the issue mentions "osallistujamäärä" though, which is wrong, sorry!